ibis/backends/dask/execution/util.py

```diff
@@ -86,6 +165,13 @@ def safe_concat(dfs: List[Union[dd.Series, dd.DataFrame]]) -> dd.DataFrame:
 
     TODO - performance.
```

Comment:
Can you expand this TODO? Not clear what performance are you referring to here

Reply:
Just general performance - this method works in that it produces the correct result, but if we start encountering performance issues this is definitely an area I would look at optimizing.

Reply:
kk, it'd be great if you can document what performance issue this function has. Just looking at this comment I have little idea why this function has performance issues.

Reply:
Added more specificity to the note.